core/services/job/spawner.go

```diff
@@ -29,21 +28,21 @@ type (
 		CreateJob(ctx context.Context, spec Job, name null.String) (Job, error)
 		DeleteJob(ctx context.Context, jobID int32) error
 		ActiveJobs() map[int32]Job
+
+		// NOTE: Only used in testing to start a job previously manually injected into DB
+		StartService(spec Job) error
```

Comment:
Not true, seems like its used in this file further down

Reply:
Yeah it's used inside the spawner and it was a private method, I had to expose it to fix a keeper test in https://github.com/smartcontractkit/chainlink/commit/34ba38379d9df2c923bfdc8a5f028a45c0969cac. Ideally I'd like to avoid code relying on this function though, especially outside of tests